examples/react/w3console/src/share.tsx

```diff
@@ -80,9 +112,43 @@ export function SpaceShare (): JSX.Element {
             value={value}
             onChange={onChange}
           />
-          <a className='w3ui-button text-center block w-52 opacity-30' style={{ opacity: downloadUrl !== '' ? '1' : '0.2' }} href={downloadUrl ?? ''} download={downloadName(downloadUrl !== '', value)}>Download UCAN</a>
+          <a className='w3ui-button text-center block w-52' style={{ opacity: downloadUrl !== '' ? '1' : '0.2' }} href={downloadUrl ?? ''} download={downloadName(downloadUrl !== '', value)}>Download UCAN</a>
         </form>
       </div>
+      <div className='mt-16 py-16 border-t border-gray-700'>
+        <Header>Import a space</Header>
+        <p className='mb-2'>Copy and paste your DID to your friend</p>
+        <div className='bg-opacity-10 bg-white font-mono text-sm py-2 px-3 rounded break-words max-w-4xl'>
+          {agent?.did()}
+        </div>
+        <div className='mt-8'>
+          <label className='w3ui-button text-center block w-52'>
+            Import UCAN
+            <input type='file' accept='.ucan,.car,application/vnd.ipfs.car' className='hidden' onChange={(e: ChangeEvent<HTMLInputElement>) => { void onImport(e) }} />
+          </label>
+        </div>
+        {proof !== undefined && (
+          <div className='mt-4 pt-4'>
+            <Header>Added</Header>
+            <div className='max-w-3xl border border-gray-700 shadow-xl'>
+              {proof.capabilities.map((cap, i) => (
+                <figure className='p-4 flex flex-row items-start gap-2' key={i}>
+                  <DidIcon did={cap.with} />
+                  <figcaption className='grow'>
+                    <a href='#' onClick={() => viewSpace(cap.with)} className='block'>
+                      <span className='block text-xl font-semibold leading-5 mb-1'>{proof.facts.at(i)?.space.name ?? 'Untitled Space'}</span>
+                      <span className='block font-mono text-xs text-gray-500 truncate'>{cap.with}</span>
+                    </a>
+                  </figcaption>
+                  <div>
+                    <a href='#' className='font-sm font-semibold' style={{ verticalAlign: '-8px' }} onClick={() => viewSpace(cap.with)}>View</a>
```

Comment:
this one too, tho I'm not wedded to the division of props here so feel free to go a different way

also better to go with `align-[-8px]` rather than setting style manually: https://tailwindcss.com/docs/vertical-align#using-custom-values
```suggestion
                    <a href='#' className='font-sm font-semibold align-[-8px]' 
                       onClick={() => viewSpace(cap.with)}>
                      View
                    </a>
```